packages/appeals-service-api/__tests__/developer/users.test.js

```diff
@@ -202,6 +202,21 @@ describe('Users', () => {
 		expect(userResponse.body.enabled).toBe(true);
 	});
 
+	it('should set status flag as added for user added by admin', async () => {
+		// given a non-admin user is added
+		const email = 'testuser1@planninginspectorate.gov.uk';
```

Comment:
The doamin for test emails should be @exmaple.com just as an extra guard rail to be avoid ever sending out emails when running tests to a genuine domain and because @example.com is a reserved doamin commonly used for documentation etc

Reply:
will check, but I think the domain has to be planninginspectorate.gov.uk for the integration test to work properly

Reply:
Cool. In the short term if this a bigger change than just changing that email address then don't worry about it. We'll just flag it as a dev task.

Reply:
fixed it - all domains are example.com on this test now